packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -94,6 +94,15 @@ describe('KeyringController', () => {
           },
         );
       });
+
+      it('should emit accountAdded', async () => {
+        await withController(async ({ controller, messenger }) => {
+          const listener = sinon.spy();
```

Comment:
I know that the current state of the `KeyringController.test.ts` file is confusing as we both use `sinon` and `jest`, but would be better to start using jest at least for new test cases